### PR TITLE
change pivot download file from xls to xlsx

### PR DESCRIPTION
**Pull request by msh-odoo**
Task:https://www.odoo.com/web#id=2127388&action=333&active_id=1251&model=project.task&view_type=form&menu_id=4720

Pad: https://pad.odoo.com/p/r.4fde01e8f05ca7b57c03c22600d70e41

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/controllers/pivot.py [hunk 1]
```diff
@@ -71,6 +74,8 @@ def export_xls(self, data, token):
                     worksheet.write(y, x+i, '', header_plain)
                 x = x + (2 * origin_count - 1)
             x, y = 1, y + 1
+            # set minimum width to cells to 20 which around 110px
+            worksheet.set_column(0, len(measure_headers), 16)
```
Comment (aab-odoo) [inline addons/web/controllers/pivot.py]: comment is outdated

**Comment by aab-odoo** [comment]
@msh-odoo Could you update the comment and rebase? Thanks.

File: addons/web/controllers/pivot.py [hunk 2]
```diff
@@ -71,6 +74,8 @@ def export_xls(self, data, token):
                     worksheet.write(y, x+i, '', header_plain)
                 x = x + (2 * origin_count - 1)
             x, y = 1, y + 1
+            # set minimum width to cells to 20 which around 88px
```
Comment (aab-odoo) [inline addons/web/controllers/pivot.py]: comment still makes no sense

Comment (msh-odoo) [reply addons/web/controllers/pivot.py]: @aab-odoo Sorry, I did change in hurry, I fixed it now

**Comment by aab-odoo** [comment]
robodoo rebase-ff

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**